CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 ## Unreleased
 
+- Remove deprecated config properties in favor of the new names (#10349):
+  - `otel.instrumentation.kafka.client-propagation.enabled` ->
+    `otel.instrumentation.kafka.producer-propagation.enabled`
```

Comment:
could you also update https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/89ee25f4d2de6a43425df7214e46fb47e63ecd79/instrumentation/kafka/README.md?plain=1#L6